src/main/java/by/exadel/internship/controller/LocationController.java

```diff
@@ -0,0 +1,31 @@
+package by.exadel.internship.controller;
+
+import by.exadel.internship.dto.locationDTO.CityDTO;
+import by.exadel.internship.dto.locationDTO.CountryDTO;
+import by.exadel.internship.service.LocationService;
+import io.swagger.annotations.Api;
+import io.swagger.annotations.ApiOperation;
+import lombok.RequiredArgsConstructor;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+
+@RestController
+@RequiredArgsConstructor
+@RequestMapping("/location")
+@Api(tags = "Endpoints for Location")
+public class LocationController {
+    private final LocationService locationService;
+
+    @GetMapping("/countries")
+    @ApiOperation("return list of countries")
+    public List<CountryDTO> getAllCountries() {
+        return locationService.getAllCountries();
+    }
+
+    @GetMapping("/cities/country-name-cities")
```

Comment:
`country-name-cities` should be part of REST path. Check [article](https://medium.com/@mwaysolutions/10-best-practices-for-better-restful-api-cbe81b06f291)

```suggestion
    @GetMapping("/cities")
```

Reply:
Done